webpack.config.js

```diff
@@ -124,6 +129,7 @@ let config = {
             },
         }),
         new ForkTsCheckerWebpackPlugin(),
+        new BundleAnalyzerPlugin(),
```

Comment:
This thing nice however I would to open this thing by default `openAnalyzer: false`

Reply:
DONE